JASP-Tests/R/tests/testthat/test-reliabilityanalysis.R

```diff
@@ -60,10 +60,18 @@ test_that("Reverse scaled items match", {
   options$sdScale <- TRUE
   
   results <- jasptools::run("ReliabilityAnalysis", "Fear of Statistics.csv", options)
-  table <- results[["results"]][["reliabilityScale"]][["data"]]
+  table <- results[["results"]][["scaleTable"]][["data"]]
   expect_equal_tables(table,
     list("scale", 0.820836210468446, 0.813045844410605, 0.724966918842779,
          0.708529526625945, 0.368244389782582, 3.08727148969273, 0.393585959365817,
          0.81017314606981, 0.831119838017159)
   )
+})
+
+test_that("Analysis handles errors - Infinity", {
+  options <- jasptools::analysisOptions("ReliabilityAnalysis")
+  options$variables <- c("debInf")
+  results <- jasptools::run("ReliabilityAnalysis", "test.csv", options)
+  errorMsg <- results[["results"]][["errorMessage"]]
+  expect_is(errorMsg, "character")
```

Comment:
Same comment as I made earlier; check for `"validationError"` in `results[["status"]]`